### PR TITLE
create proposals app

### PR DESCRIPTION
**Pull request by isabella-karabasz**
Created blank proposals app

### CODE REVIEW

**Comment by isabella-karabasz** [comment]
@aalmiramolla @serfer2 Pull request created, yet somehow I can't seem to add reviewers

File: meetupselector/settings/default.py [hunk 1]
```diff
@@ -46,7 +46,8 @@
 
 OUR_APPS = [
     "meetupselector.user",
-    "meetupselector.talks",
+    "meetupselector.talks", 
+    "meetupselector.proposals"
```
Comment (serfer2) [inline meetupselector/settings/default.py:50]: As a good practice, it is recommended to add a trailing comma at the end of multi-line dictionaries, lists, tuples ...
I mean:
```python
OUR_APPS = [
    "meetupselector.user",
    "meetupselector.talks",
    "meetupselector.talks", 
    "meetupselector.proposals",
]
```
Some lints would raise an error when missing trailing comma.

File: meetupselector/proposals/views.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```
Comment (serfer2) [inline meetupselector/proposals/views.py:1]: Same as `tests.py`, this file can be deleted.

File: meetupselector/proposals/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (serfer2) [inline meetupselector/proposals/tests.py:1]: I think we can safely delete this file. It is created by default by django but in our case we're not using it.

**Comment by serfer2** [comment]
@isabella-karabasz I left some non-blocking minor comments. Feel free to merge whenever you want. Well done! 😃

File: meetupselector/settings/default.py (hunk 1 above)
Comment (isabella-karabasz) [reply meetupselector/settings/default.py:50]: Good to know, thanks!

**Pull request merged**